### PR TITLE
Swift: add CFG and PrintAst consistency queries, enabling them in CI

### PR DESCRIPTION
**Pull request by redsun82**

### CODE REVIEW

File: swift/ql/src/consistency/PrintAstConsistency.ql [hunk 1]
```diff
@@ -0,0 +1 @@
+import codeql.swift.printast.Consistency
```
Comment (aibaars) [inline swift/ql/src/consistency/PrintAstConsistency.ql:1]: I think we should not include these queries in the `codeql/swift-queries` pack. They are not meant for users. Other languages seem to store them in separate packs `LANG/ql/consistency-queries`.

**Review by aibaars — CHANGES_REQUESTED** [review]

File: swift/ql/src/consistency/PrintAstConsistency.ql (hunk 1 above)
Comment (redsun82) [reply swift/ql/src/consistency/PrintAstConsistency.ql:1]: makes sense!

Comment (redsun82) [reply swift/ql/src/consistency/PrintAstConsistency.ql]: done

**Comment by redsun82** [comment]
I discussed a plan with @MathiasVP:
* we don't need to fix these inconsistencies for the release, but we should check our latest PRs did not introduce any new inconsistencies
* we should turn on this CI check with all expected results on main (I'll finish this tomorrow)
* we should open issues to fix these inconsistencies

The current status of this PR is highly incomplete, also the fix to the property wrapper double needs to be revisited, as it turns out the initializer is not _always_ shared between a var decl and its synthesized property wrapper backing binding.

File: swift/ql/lib/codeql/swift/controlflow/internal/ControlFlowGraphImpl.qll [hunk 2]
```diff
@@ -969,6 +959,9 @@ module Decls {
         result.asAstNode() = ast.getPattern(j).getFullyUnresolved()
       )
       or
+      // PropertyWrapperBackingVarBinding is a special case as it shares the init
+      // with the original variable binding (see also PrintAstNode.qll)
+      ast != any(VarDecl d).getPropertyWrapperBackingVarBinding() and
```
Comment (MathiasVP) [inline swift/ql/lib/codeql/swift/controlflow/internal/ControlFlowGraphImpl.qll:964]: As we discussed in our sync. See https://codeql.github.com/docs/ql-language-reference/formulas/#equality.

```suggestion
      not ast = any(VarDecl d).getPropertyWrapperBackingVarBinding() and
```

**Pull request closed without merging**